### PR TITLE
[Review] added convention about module paths

### PR DESCRIPTION
**Pull request by juliankrispel**

### CODE REVIEW

**Comment by juliankrispel** [comment]
cc @rainforestapp/customer-product

File: Style.md [hunk 1]
```diff
@@ -98,3 +98,11 @@ Example: (__AVOID__)
 - Never nest ternaries.
 - Double-quotes for 'static' react propTypes like `<something some="hello"/>`.
 - Backtick quotes for string concatenation, also in jsx: ```<something some={`${a}_${b}`}/>```
+
+
+## Paths & Module references
+Only use absolute paths and avoid the use of '../../'.
```
Comment (ziahamza) [inline Style.md]: how about `./actions`?

Comment (prayogoa) [inline Style.md]: what about tests, they usually need to do `../`.
Modules in the same folder also uses `./`. They're meant to be coupled together and it makes little sense if it breaks when you move the folder around.

**Comment by juliankrispel** [comment]
ok makes sense @prayogoa @ziahamza, I'll update the pr

**Pull request closed without merging**